frontend/public/co-fetch.js

```diff
@@ -154,7 +153,14 @@ export const coFetchCommon = (url, method = 'GET', options = {}, timeout) => {
     if (response.headers.get('Content-Type') === 'text/plain') {
       return response.text();
     }
-    return response.json();
+    // Cypress tests throwing "Unexpected end of JSON input" during response.json(), trapping and returning
```

Comment:
We need to figure out what's happening in the tests here instead of adding this code to co fetch

Reply:
I am still working on this.

Reply:
Hi @spadgett, is this a PR blocker?  Can I file a bug and work on this in a follow up PR?

Reply:
> Hi @spadgett, is this a PR blocker? Can I file a bug and work on this in a follow up PR?

This is a PR blocker in my opinion

Reply:
Hi @spadgett, I reverted `co-fetch.js`, and cypress tests all pass.  I don't know what changed in the pipeline/infrastructure where we no longer get "Unexpected end of JSON input" when expiring a silence, but kinda makes sense as the monitoring test was the first one I did, which had been passing for weeks, then suddenly was getting this JSON error, now just as suddenly seems to no longer be an issue.
I pushed the restored `co-fetch.js` and CI failed a number of times (locally passed 5+ times) with seemingly unrelated, but no less concerning failures (timeouts, failed to click a button clearly shown in the screenshot, etc..).  Last CI passed, but I'm concerned about the persistent flakes.